### DISCUSSION TITLE
Svelte radix is not adding new components

### DISCUSSION THREAD

**Discussion by JonathonRP** [huntabyte/shadcn-svelte#124] [General]
The maintainer of svelte radix is not adding now components but spending time on his own melt-ui, just curious if the maintainers are aware and have a plan for this?

**Comment by braden-w** [comment]
^ Saw [the announcement](https://github.com/radix-svelte/radix-svelte/blob/develop/ANNOUNCEMENT.md) on Radix Svelte website and the [Melt UI](https://www.melt-ui.com/docs/builders/dialog) website. APIs seem similar but seems more Svelty. Would it make sense for the project to move over to Melt UI or other alternatives?

**Comment by riziles** [comment]
@braden-w & @JonathonRP ,

Please see <https://github.com/huntabyte/shadcn-svelte/issues/95#issuecomment-1595350269>. Agreed that this notice should be displayed more prominently. 

@huntabyte is a pretty active contributor to Melt UI. Very much looking forward to that YouTube video....
